packages/client/shared/gqlIds/TeamMemberId.ts

```diff
@@ -1,8 +1,8 @@
 
 const TeamMemberId = {
-  join: (teamId: string, userId: string) => `${userId}:${teamId}`,
+  join: (teamId: string, userId: string) => `${userId}::${teamId}`,
   split: (id: string) => {
-    const [userId, teamId] = id.split(':')
+    const [userId, teamId] = id.split('::')
```

Comment:
this helper isn't used anywhere (yet) probably because it was buggy.
we'll use this in the future to replace `toTeamMemberId` and `fromTeamMemberId`